core-api/src/main/java/com/optimizely/ab/odp/ODPApiManager.java

```diff
@@ -0,0 +1,22 @@
+/**
+ *    Copyright 2022, Optimizely Inc. and contributors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.optimizely.ab.odp;
+
+import java.util.List;
+
+public interface ODPApiManager {
+    String fetchQualifiedSegments(String apiKey, String apiEndpoint, String userKey, String userValue, List<String> segmentsToCheck);
```

Comment:
It looks like you plan to open all ODP APIs here to public for customization support. It looks good for now and we can continue discussion the final apis.

Reply:
Yes, this will contain all the methods that make an external call. ODPEvents calls will also go here